src/main/CMakeLists.txt

```diff
@@ -311,12 +311,15 @@ if (OS_IS_WIN)
     install(FILES
             ${PROJECT_SOURCE_DIR}/build/qt.conf
             ${QtInstallLibraries}
-            ${QT_INSTALL_BINS}/libEGL.dll
-            ${QT_INSTALL_BINS}/libGLESv2.dll
```

Comment:
Sure those are no longer needed? Or just not on your machine with your graphics card/driver?

Reply:
(See https://github.com/musescore/MuseScore/commit/8e7a14e5743f580258e8f3e6803153e53228ff77#commitcomment-72425448)